crates/sui-core/src/consensus_handler.rs

```diff
@@ -200,8 +201,18 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
                 }
             };
 
+            let certificate_author = AuthorityName::from_bytes(
+                self.committee
+                    .authority_safe(&output_cert.header.author)
+                    .protocol_key_bytes()
+                    .0
+                    .as_ref(),
+            )
+            .unwrap();
```

Comment:
@andll my understanding is that by doing just simple `let certificate_author = AuthorityName = self.committee.authority_safe(&output_cert.header.author).protocol_key().into()` would do the trick as well and we would pay the cost of converting the `PublicKey` to `bytes` once - as I can see it gets via `[OnceCell](https://github.com/MystenLabs/fastcrypto/blob/c2f79b1807bff7d09517b631191b61f2614c641c/fastcrypto/src/bls12381/mod.rs#L137) . To play it safe here I preferred to have the keys pre-converted to bytes and provide via `protocol_key_bytes` , but might be redundant